Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -814,6 +814,8 @@
     sprite: Objects/Specific/Hydroponics/corn.rsi
   - type: Produce
     seedId: corn
+  - type: Extractable
+    grindableSolutionName: food
```

Comment:
Move this down a line. The test failure is that there's two ExtractibleComponents on this entity.

Reply:
Got it. Move the grindable down under the second type: extractable, and delete the first one? That will work?